<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PIL 2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="Paint">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="MainTools">
       <attribute name="title">
        <string>Основные инструменты</string>
       </attribute>
       <widget class="QLabel" name="picture1">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>-10</y>
          <width>570</width>
          <height>590</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>311</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="ClockwiseRotation">
           <property name="text">
            <string>По часовой стрелке</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CounterclockwiseRotation">
           <property name="text">
            <string>Против часовой стрелки</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>469</y>
          <width>321</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="OneStepBack1">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Back">
           <property name="text">
            <string>Отменить все</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>191</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSlider" name="DegreeChanger">
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="VisibilitySlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>84</y>
          <width>120</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Поворот</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Прозрачность</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="logo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>301</width>
          <height>161</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>209</y>
          <width>311</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="Fading">
           <property name="text">
            <string>Блеклость</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="Contrast">
           <property name="text">
            <string>Контраст</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="LessBright">
           <property name="text">
            <string>Понизить яркость</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="MoreBright">
           <property name="text">
            <string>Повысить яркость</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Filters">
       <attribute name="title">
        <string>Фильтры</string>
       </attribute>
       <widget class="QLabel" name="picture2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>-10</y>
          <width>570</width>
          <height>590</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>311</width>
          <height>241</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="Filters_2">
         <item row="1" column="1">
          <widget class="QPushButton" name="BlackWhiteNoGray">
           <property name="text">
            <string>Ч/Б без серого</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="Sharpness">
           <property name="text">
            <string>Резкость</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="Inversion">
           <property name="text">
            <string>Инверсия</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="Smoothing">
           <property name="text">
            <string>Сглаживание</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="BlackWhite">
           <property name="text">
            <string>Ч/Б</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="Borders">
           <property name="text">
            <string>Контур</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="Random">
           <property name="text">
            <string>Рандом</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="OneStepBack2">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>71</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>B</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>240</y>
          <width>241</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="Sliders_2">
         <item>
          <widget class="QSlider" name="RedSlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="GreenSlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="BlueSlider">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="pageStep">
            <number>15</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>15</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>359</y>
          <width>311</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="Party">
           <property name="text">
            <string>Вечеринка</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="ItIsPouring">
           <property name="text">
            <string>Дождливый вечер</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
